src/Algebra/Graph/AdjacencyIntMap/Algorithm.hs

```diff
@@ -122,15 +98,15 @@ bfsForestFrom' vs g = evalState (explore vs) IntSet.empty where
 -- bfs [2] ('edge' 1 2)                                   == [[2]]
 -- bfs [3] ('edge' 1 2)                                   == []
 -- bfs [1,2] ('edge' 1 2)                                 == [[1],[2]]
--- bfs [2,1] ('edge' 1 2)                                 == [[2],[1]]
+-- bfs [2,1] ('edge' 1 2)                                 == [[2,1]]
 -- bfs [1,2] ( (1*2) + (3*4) + (5*6) )                  == [[1],[2]]
```

Comment:
This should probably be `[[1,2]]` too?

Reply:
At the top level, the vertices aren't rearranged, so 2 comes before 1 here. 

They do get nubbed at the top leve in `filterM discover [ v | v <- hasVertex v g ]`, possibly it makes sense to present them in ascending order from that? My thinking was that it may be nice to retain control over the order the search trees get expanded.

Reply:
In tests you still have:

```haskell
bfs [1,2] ((1*2) + (3*4) + (5*6))    == [[1,2]]
```